src/features/isanteplus/patientCoversheet.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Patient Summary
+
+ Background:
+    Given system user logs into Isanteplus application and goes to the Home page
+
+  @patientCoverSheet
+  Scenario: Patient Cover sheet
+  When Search for and select Patient "<patientName>"
+  Then  Selected patient’s ‘Cover Sheet’ will be displayed with the following
```

Comment:
make this more meaning full.
Atleast say ``Selected patient’s ‘Cover Sheet’ will be displayed with all the right details`

Reply:
Resolved thnks